critiquebrainz/frontend/external/musicbrainz_db/place.py

```diff
@@ -23,28 +23,43 @@ def get_place_by_id(mbid):
     key = cache.gen_key(mbid)
     place = cache.get(key)
     if not place:
-        place = _get_place_by_id(
-            mbid, includes=['artist-rels', 'place-rels', 'release-group-rels', 'url-rels'],
-        )
+        place = fetch_multiple_places(
+            mbids=[mbid],
+            includes=['artist-rels', 'place-rels', 'release-group-rels', 'url-rels'],
+        ).get(mbid)
     cache.set(key=key, val=place, time=DEFAULT_CACHE_EXPIRATION)
     return place
 
 
-def _get_place_by_id(place_id, includes=[]):
+def fetch_multiple_places(*, mbids, includes=None):
+    """Get info related to multiple places using their MusicBrainz IDs.
+
+    Args:
+        mbids (list): List of MBIDs of places.
+        includes (list): List of information to be included.
+
+    Returns:
+        Dictionary containing info of multiple places keyed by their mbid.
+    """
+    includes = dict() if includes is None else includes
     includes_data = {}
     check_includes('place', includes)
     with mb_session() as db:
         query = db.query(models.Place)
-        place = get_something_by_gid(query, models.PlaceGIDRedirect, place_id)
-        if not place:
-            raise mb_exceptions.NoDataFoundException("Couldn't find a place with id: {place_id}".format(place_id=place_id))
+        places = []
+        for mbid in mbids:
+            place = get_something_by_gid(query, models.PlaceGIDRedirect, mbid)
+            if not place:
+                raise mb_exceptions.NoDataFoundException("Couldn't find a place with id: {mbid}".format(mbid=mbid))
+            places.append(place)
+        place_ids = [place.id for place in places]
 
         if 'artist-rels' in includes:
-            includes_data.setdefault('relationship_objs', {})['artist-rels'] = entity_relation_helper(db, 'artist', 'place', place.id)
+            entity_relation_helper(db, 'artist', 'place', place_ids, includes_data)
```

Comment:
This is a little complex to understand - I think that you're passing in `includes_data` as an argument and it's getting populated in `get_relationship_links`. Is that correct?
If that is the case, perhaps we shouldn't call the function `get_.....`. The documentation should be updated to say that this value is altered, instead of being passed in as an argument.